### DISCUSSION TITLE
Création de la zone Discussions

### DISCUSSION THREAD

**Discussion by AnonymeUSB** [AnonymeUSB/CPWJP#1] [Annonces]
C'est en ce jour que fût créer la zone discussions. Voilà son but : 

- Diffuser des annonces.
- Centralisez les discussions.
- Vous fournir de l'aide en cas de bug.
-  Posez vos questions.
-  Donnez vos idées de futur projets.

Sur ce, profitez bien de cette zone discussion !

**Comment by clecle-123** [comment]
hey people
